### PR TITLE
Footer CTA for Road to React Native

### PR DESCRIPTION
**Pull request by MatthysDev**
✅ Closes: #37

### CODE REVIEW

File: libs/ui/src/lib/footer-book/footer-book.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            style="w-44 my-4"
+            href="https://gumroad.com/l/road-react-native"
+          >
+            Check it out !
+          </Button>
+        </div>
+        <div className="w-1/3 flex-initial">
+          <div className="-rotate-3 p-4">
+            <img
+              src="https://raw.githubusercontent.com/flexbox/next.davidl.fr/master/public/images/road-to-react-native-book.jpg?token=GHSAT0AAAAAABZPZCRZKZTQG6VO2WN7FWW4Y5BSNQA"
```
Comment (flexbox) [inline libs/ui/src/lib/footer-book/footer-book.tsx:29]: Do not hotlink content here 

1. download the image
2. put it in `/public` folder
3. use Nextjs `Image` component

**Review by flexbox — CHANGES_REQUESTED** [review]

**Comment by flexbox** [comment]
@MatthysDev where is the storybook preview?

File: libs/ui/src/lib/footer-book/footer-book.tsx (hunk 1 above)
Comment (MatthysDev) [reply libs/ui/src/lib/footer-book/footer-book.tsx:29]: There is no `/public`in weshipit-ui

Comment (flexbox) [reply libs/ui/src/lib/footer-book/footer-book.tsx:29]: My bad if it works it workd 😅

**Pull request merged**